src/test/java/com/wiley/autotest/framework/pages/TestElementPage.java

```diff
@@ -0,0 +1,43 @@
+package com.wiley.autotest.framework.pages;
+
+import com.wiley.autotest.selenium.context.AbstractPage;
+import org.openqa.selenium.By;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by shekhavtsov on 20/07/2017.
+ */
+@Component
+public class TestElementPage extends AbstractPage {
+
+    public TestElementPage checkFirstVisibleElement() {
+        element(By.cssSelector("li")).should().beDisplayed();
+        element(By.cssSelector("li")).should().haveText("element #1 (visible)");
+        return this;
+    }
+
+    public TestElementPage checkElementBecomeVisible() {
+        element(By.id("waitElement")).should().beDisplayed();
+        return this;
+    }
+
+    public TestElementPage checkAnimationElement() {
+        element(By.id("myAnimation")).should().beDisplayed();
+        element(By.id("myAnimation")).should().beDisplayed();
+        element(By.id("myAnimation")).should().beDisplayed();
```

Comment:
Not sure i understand the idea of this particular test? What does it really check and in what situation do you expect it to fail? I think the best way would be to provide a comment to this method - explaining the idea of it